bedrock/mozorg/views.py

```diff
@@ -422,7 +422,7 @@ def new_home(request, locale=None):
 
 def home(request):
```

Comment:
I'm not 100% sure either. :smile: But as far as I can tell, there's no longer a reason to have 2 views and 1 class view. Should just need a single view. @jgmize - Care to weigh in?

If we can/do remove `HomeTestView`, some tests will need updating.